homeassistant/components/lightwave/__init__.py

```diff
@@ -6,16 +6,24 @@
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.discovery import async_load_platform
 
-LIGHTWAVE_LINK = "lightwave_link"
-
+CONF_SERIAL = "serial"
+CONF_TRV = "trv"
+CONF_TRVS = "trvs"
+DEFAULT_PORT = 7878
+DEFAULT_IP = "127.0.0.1"
 DOMAIN = "lightwave"
+LIGHTWAVE_LINK = "lightwave_link"
+LIGHTWAVE_TRV_PROXY = "lightwave_proxy"
+LIGHTWAVE_TRV_PROXY_PORT = "lightwave_proxy_port"
+PROXY_IP = "proxy_ip"
+PROXY_PORT = "proxy_port"
```

Comment:
All config keys start with `CONF_`, therefore rename them to `CONF_PROXY_IP` and `CONF_PROXY_PORT`

Reply:
Done